vector/src/main/java/im/vector/app/features/call/VectorCallViewModel.kt

```diff
@@ -147,6 +147,7 @@ class VectorCallViewModel @AssistedInject constructor(
                     setState { copy(otherKnownCallInfo = null) }
                 }
             }
+            _viewEvents.post(VectorCallViewEvents.StopScreenSharingService)
```

Comment:
Here I think the set of the `isSharingScreen` to false in the state is missing. Maybe we should reuse an mutualize what is done in `handleToggleScreenSharing()` when screen is shared:
```
if (isSharingScreen) {
    call?.stopSharingScreen()
    setState {
        copy(isSharingScreen = false)
    }
    _viewEvents.post(VectorCallViewEvents.StopScreenSharingService)
}
```
What do you think?

Reply:
Done, although we finish the call activity as soon as the call ended.